src/PKSim.Core/Snapshots/AdvancedParameter.cs

```diff
@@ -2,6 +2,7 @@
 {
    public class AdvancedParameter : ParameterContainerSnapshotBase
    {
+      public int Seed { get; set; }
```

Comment:
See. Not saved to snapshot :)

Reply:
To be discussed. Will create an issue for 7.4 for this.